tex/rules.tex

```diff
@@ -270,16 +270,33 @@ \subsection{Goalie \label{ref-008}}
 
 \subsection{Pushing \label{ref-pushing}}
 
-Within the penalty area, the goalie has priority. Attacking robots are not
-supposed to push the goalie in any way.
-
-If the attacker and the goalie touch each other and at least one of them has
-physical contact with the ball, the ball will be moved to the nearest
-unoccupied neutral spot immediately.
+Robots are not allowed to push each other at any time and at any place.
+This applies to two robots of the same team as well as two robots of the competing opponents.
+The robots may use vision, touch sensors attached to avoidance rings, or any other method
+to detect the avoidance rings of other robots, see section \ref{ref-avoidance}.
+A robot is allowed to play only if it can reliably detect touching or coming close to 
+another robot in all directions of its own movement. When the robot detects another
+robot in any particular direction, it must stop all movement in that direction immediately.
+If any robot happens to push any other robot because it has failed to detect the other robot,
+it is removed from play for 30 seconds by a referee.
```

Comment:
There are other parts of the rules where the robot is removed from play (namely "Out of Bounds" and "Damaged" robots), but in those cases they are removed for 60 seconds.

Is there any specific reason for choosing 30 in this case? Having the same amount of time in all cases would make it a bit easier on the referees, as they would not need to track different time intervals.